src/Raven.Server/Documents/Handlers/Admin/AdminConfigurationHandler.cs

```diff
@@ -18,7 +18,7 @@ namespace Raven.Server.Documents.Handlers.Admin
 {
     public class AdminConfigurationHandler : DatabaseRequestHandler
     {
-        [RavenAction("/databases/*/admin/configuration/settings", "GET", AuthorizationStatus.DatabaseAdmin)]
+        [RavenAction("/databases/*/admin/configuration/settings", "GET", AuthorizationStatus.DatabaseAdmin, IsDebugInformationEndpoint = true)]
```

Comment:
I added only this endpoint and not "/admin/configuration/settings" endpoint because the second defined as `AuthorizationStatus.ClusterAdmin` and the higher `AuthorizationStatus` of the "debug-info" endpoint is `AuthorizationStatus.Operator`.
But actually it seems (for now at least) that the "/admin/configuration/settings" endpoint contains less information.
In the current implementation, the user will not be possible to get the settings if there are no databases but will get duplicate of the server setting on any database.
In addition, "/databases/*/debug/info-package" endpoint defined as `AuthorizationStatus.ValidUser` so should we ignore the setting in the debug package if the user has less permission then `AuthorizationStatus.DatabaseAdmin` as "/databases/*/admin/configuration/settings" defined?